### PR TITLE
Adding "connection-name" optional parameter

### PR DESCRIPTION
**Pull request by bluedenim**
Updating on poetry installation steps, README.md, and requirements to support Django 1.11+

Inspired by https://github.com/bluedenim/vmigration-helper/pull/14

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
+FROM python:3.9
 
 WORKDIR /code
 
 RUN pip install --upgrade pip
 
 # Install and setup Poetry
-RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python -
-RUN echo export PATH=\$PATH:\$HOME/.poetry/bin >> /root/.bashrc
+RUN curl -sSL https://install.python-poetry.org | python3 -
```
Comment (bluedenim) [inline Dockerfile:8]: Let's hope their install script is stable now.

File: vmigration_helper/helpers/command.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+from abc import ABC
+
+from django.core.management import BaseCommand
+from django.db import DEFAULT_DB_ALIAS
+
+
+class MigrationCommand(BaseCommand, ABC):
```
Comment (bluedenim) [inline vmigration_helper/helpers/command.py:10]: Common options for all our migration commands

File: Dockerfile (hunk 1 above)
Comment (setu4993) [reply Dockerfile:8]: 😅. They say it's going to be 🤞🏽.

File: vmigration_helper/helpers/command.py (hunk 2 above)
Comment (setu4993) [reply vmigration_helper/helpers/command.py:10]: Nice clean-up of the dupe from #14.

File: vmigration_helper/helpers/command.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
+from abc import ABC
+
+from django.core.management import BaseCommand
+from django.db import DEFAULT_DB_ALIAS
```
Comment (setu4993) [inline vmigration_helper/helpers/command.py]:
```suggestion
from django.conf import settings
```

File: vmigration_helper/helpers/command.py [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+
+class MigrationCommand(BaseCommand, ABC):
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--connection-name',
+            default=DEFAULT_DB_ALIAS,
```
Comment (setu4993) [inline vmigration_helper/helpers/command.py:22]:
```suggestion
            default=settings.DEFAULT_DB_ALIAS,
```

File: requirements.txt [hunk 5]
```diff
@@ -1,3 +1,3 @@
 asgiref==3.4.1
-Django==4.0
+Django==1.11.29
```
Comment (setu4993) [inline requirements.txt]: Curious why we need a `requirements.txt` in addition to the `pyproject.toml` + `poetry.lock`?

**Review by setu4993 — APPROVED** [review]
Thanks, Van! Left 1 note about using `django.conf.settings.DEFAULT_DB_ALIAS` instead of `django.db.DEFAULT_DB_ALIAS` to correctly match the default behavior with the docs. Rest looks great.

Thanks for cleaning up and making it nicer! Don't forget to version bump at the end!

File: requirements.txt (hunk 5 above)
Comment (bluedenim) [reply requirements.txt]: We don't need it. It's leftover.

File: vmigration_helper/helpers/command.py [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+    This class provides a place for common concerns to all migration commands to be implemented. For instance, it
+    registers the "connection-name" optional parameter to the command and surfaces the parameter as
+    the "connection_name" property.
+    """
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--connection-name',
+            default=DEFAULT_DB_ALIAS,
+            help=f'The connection to use for migration commands. Defaults to settings.DEFAULT_DB_ALIAS'
```
Comment (setu4993) [inline vmigration_helper/helpers/command.py]: Nice, I like this clarification.

**Review by setu4993 — APPROVED** [review]
Looks great, thanks Van!

**Pull request merged**